### DISCUSSION TITLE
KotlinUnreal could not be compiled. Try rebuilding from source manually.

### DISCUSSION THREAD

**Discussion by IvanEOD** [hiperbou/kotlin-unreal#26] [Q&A] [answered]
I am trying to get setup to try this out, I am getting an error: 

The following modules are missing or built with a different game version: 
    Kotlin Unreal 
Would you like to rebuild them now? 

When I say yes, it tells me "KotlinUnreal could not be compiled. Try rebuilding from source manually."

I am using UE 5.0.3.
Any help would be greatly appreciated!

**Comment by hiperbou** [comment, marked as answer]
Make sure you are following the installation instructions on the Wiki, First of all, make sure to installl the Unreal.js plugin.

https://github.com/hiperbou/kotlin-unreal/wiki/Installation

If you still get the same error, try opening the KotlinUnreal.sln project in Visual Studio and build from there. It should give you a better error message to understand what's happening.

I haven't updated to UE 5.0.3 yet, but I could try it tomorrow if you're still having problems and try to fix it.

**Reply by IvanEOD** [reply]
I will try this, I didn't think of trying to redo the build in Visual Studio. I will update ASAP.

**Reply by hiperbou** [reply]
I have updated to UE 5.0.3, downloaded this repository and the game launched without issues after rebuilding.
Make sure you are able to create a C++ Project in Unreal, to ensure you have all required dependencies installed.
Try to create a project with the Third Person Template with C++ and make sure you are able to build and run it.
